src/app/domain/backup/services/uploadMedias.ts

```diff
@@ -0,0 +1,196 @@
+/* eslint-disable promise/always-return */
+import Minilog from '@cozy/minilog'
+import { CameraRoll } from '@react-native-camera-roll/camera-roll'
+import { Platform } from 'react-native'
+import RNBackgroundUpload, {
+  UploadOptions
+} from 'react-native-background-upload'
+import RNFileSystem from 'react-native-fs'
+
+import { setMediaAsBackuped } from '/app/domain/backup/services/manageLocalBackupConfig'
+import { Media } from '/app/domain/backup/models/Media'
+
+import type CozyClient from 'cozy-client'
+
+const log = Minilog('💿 Backup')
+
+interface UploadMediaResult {
+  success: boolean
+  data?: object
+}
+
+export const uploadMedias = async (
+  client: CozyClient,
+  backupFolderId: string,
+  mediasToUpload: Media[]
+): Promise<boolean> => {
+  for (const mediaToUpload of mediasToUpload) {
+    try {
+      const uploadUrl = getDriveUrl(client, backupFolderId, mediaToUpload)
+
+      const { success } = await uploadMedia(client, uploadUrl, mediaToUpload)
```

Comment:
There is no error you can get from this `uploadMedia` ?

Reply:
I find the API pretty strange. Does `uploadMedia` throws if an error occurs? If yes, then why are we checking the success? 

If not, how can we have the real error logged somewhere?